cli/impl/src/main/java/io/helidon/build/cli/impl/InitCommand.java

```diff
@@ -93,6 +98,17 @@ protected void invoke(CommandContext context) throws Exception {
             assertSupportedVersion(helidonVersion);
         }
 
+        // Set system properties from a file if provided on command-line
```

Comment:
Can this be done within `InitOptions.initProperties()` ?

Reply:
`InitOptions.initProperties()` is used only inside `io.helidon.build.cli.impl.ArchetypeInvoker.V1Invoker#invoke`. Do I need to change the logic of `io.helidon.build.cli.impl.ArchetypeInvoker.V2Invoker#invoke` and use `InitOptions.initProperties()`  inside it?

Reply:
Sorry that was wrong. `props-file` needs to be a global option ; the implementation should be somewhere in the harness and the properties should be available with `context.properties` like any other property passed with `-Dkey=value` 

We shouldn't have to change the init command or the invoker.